README.md

```diff
@@ -16,6 +16,8 @@ Please climb onboard to help us set our climate data free! #Klimatkollen /#FreeC
 
 ## Get started
 
+Detailed setup instructions are available in [doc/getting-started.md](doc/getting-started.md), including how to install the dependencies.
```

Comment:
Perhaps an even clearer wording regarding the fact that these are instructions for absolute beginners?

Reply:
"For absolute beginners, step-by-step setup instructions, including how to install all necessary dependencies, can be found in [doc/getting-started.md]."
?

Reply:
I adopted some of your wording but feel averse to the "absolute beginners" label as it's subjective and not very specific.